Summer school 2020 wrap-up

jngrad opened the pull request:
Description of changes:
- remove summer school announcement
- document CMake version and installation (fixes #3946)
- fix CMake build system for tutorial 11 (partial fix for #3942, fixes #3944)
- improve testing of tutorial 04 (partial fix for  #3940)
- fix typos

jngrad committed:
tutorials: Fix typos

jngrad committed:
tutorials: Fix mathematical formula

jngrad committed:
tutorial 04: part 3: Clarify instructions

jngrad committed:
tutorial 04: part 3: Improve statistics

Check for the monomer dffusion coefficient D0 in the LD simulation.
The LB simulation is too slow to get an accurate estimate of D0.
Run the tutorial for longer when generating the HTML output.

jngrad committed:
tutorial 11: Fix CMake for HTML output

jngrad committed:
Revert "Invitation summer school 2020"

This reverts commit 2051b05a819997886550374e012c96f7bceb6efd.
The event has ended.

jngrad committed:
docs: Document ESPResSo version requirements

KaiSzuttor commented on doc/sphinx/installation.rst:
```diff
@@ -37,7 +37,7 @@ The following tools libraries, including header files, are required to be able
 
 CMake
-    The build system is based on CMake
+    The build system is based on CMake. At least version 3.11 is required.
```
Do you really want to document version requirements twice? In CMakeLists.txt and in the documentation?

KaiSzuttor replied:
i think it would be a better idea to give a short hint for various reasons if the configuration of espresso fails. Such information is prone to get outdated fast.

jngrad replied:
Latest CMake versions (3.11+) usually do a good job of telling which dependencies are not met. We could also have a FAQ on the wiki for common issues (see #3947). I've removed the version numbers in 958d35d.

RudolfWeeber commented:
``
CMake
-    The build system is based on CMake
+    The build system is based on CMake. At least version 3.11 is required.`
```

>Do you really want to document version requirements twice? In CMakeLists.txt and in the documentation?

CmakeList is no documentation :)

In this particular case, this is warranted, since a major distro currently in use ships an older CMake.

jngrad committed:
docs: Remove dependency versions

These versions occasionally get out of sync with the
CMakeLists.txt files. Only provide general information
(e.g. Python 3, MPI environment that implements MPI
standard 1.2). The detailed version numbers depend on
the target machine architecture and the enabled ESPResSo
features. For Python dependencies, see requirements.txt.
